Adjust formatting, removing blanks spaces and unneeded code.

TimothyMakkison opened the pull request:
- Fixed indenting for `DICache` declaration.
- Remove blankline if `ThrowInvalid***` is used.
- Replaces `else` statement with `return`

Should be merged after #122

martinothamar commented:
I squash merge PRs, so I think I gave you some conflicts because when merging #122 , sorry! 

Thanks for this! ready to merge this whenever conflicts are resolved

TimothyMakkison commented:
> I squash merge PRs, so I think I gave you some conflicts because when merging https://github.com/martinothamar/Mediator/pull/122 , sorry!

No worries, I expected it to happen 😄

 I've squashed this PR down to one commit and rebased it
